docs/explanations/audit-scanner/audit-scanner.md

```diff
@@ -1,89 +1,78 @@
-situations where resources already inside of the cluster are no longer
-compliant. The audit scanner feature provides Kubernetes administrators with a
-tool to consistently verify the compliance state of their clusters.
-
-To illustrate the usage of the audit scanner in Kubewarden, let's consider the
-following scenario.
-
-Assume Bob is deploying a Wordpress Pod inside of the cluster. He's new to
-Kubernetes, hence he makes a mistake and deploys this Pod running as a
-privileged container. Since there's no policy preventing that, the Pod is
-successfully created inside of the cluster.
-
-Some days later, Alice, the Kubernetes administrator, enforces a Kubewarden
-policy that prohibits the creation of privileged containers. The Pod deployed
-by Bob keeps running inside of the cluster.
-
-However, thanks to the report generated by the audit scanner, Alice can
-quickly identify all the workloads that are violating her policies; including
-the Wordpress Pod created by Bob.
-
-To make that happens, audit scanner get all resources that should be audited,
-build a fake admission request with the resource's data and send it to the
-policy server in a different endpoint exclusively used to audit requests.
-However, for the policy evaluating the request, there is no differences from a
-real or an audit request. The data received are the same. Furthermore, this
-policy server endpoint is instrumented to collect data of the evaluation as
-the one used to validate request from the control plane. Therefore, users can
-use their monitoring tools analyze this data as well.
+The `audit-scanner` component constantly checks resources in the cluster.
+It flags the ones that don't adhere with the Kubewarden policies deployed in the cluster.
+
+Policies evolve over time.
+New ones are deployed, existing ones are updated.
+Versions and configuration settings change.
+This can lead to situations where resources already inside the cluster are no longer compliant.
+The audit scanning feature provides Kubernetes administrators with a tool that constantly verifies the compliance state of their clusters.
+
+To explain the use of the audit scanner in Kubewarden, consider the following scenario.
+
+Assume Bob is deploying a WordPress Pod in the cluster.
+Bob is new to Kubernetes, makes a mistake and deploys the Pod running as a privileged container.
+At this point there's no policy preventing that so the Pod is
+successfully created in the cluster.
+
+Some days later, Alice, the Kubernetes administrator, enforces a Kubewarden policy that prohibits the creation of privileged containers.
+The Pod deployed by Bob keeps running in the cluster as it already exists.
+
+A report generated by the audit scanner lets Alice identify all the workloads that are violating creation policies.
+This includes the WordPress Pod created by Bob.
+
+The audit scanner operates by:
+
+- identifying all the resources to audit
+- for each it builds a fake admission request with the resource's data
```

Comment:
```suggestion
- for each it builds a synthetic admission request with the resource's data
```

I went with "synthetic" in the blog post because of style, I wonder if it would be a good fit here too. The only appearance of the word "fake" is here.

Reply:
Thanks!